Modules/Scripted/WebServer/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(MODULE_NAME WebServer)
 set(MODULE_PYTHON_SCRIPTS
   ${MODULE_NAME}.py
   ${MODULE_NAME}Lib/__init__
+  ${MODULE_NAME}Lib/base.py
```

Comment:
It would be more consistent to name this BaseRequestHandler.py since in this folder the name Slicer is used to refer to the handler that handles Slicer-related requests.

Reply:
My preference is not to name submodules with the same class names they implement, because it is easier to confuse new users.

This is an easy mistake to make:
```py
import WebServerLib.BaseRequestHandler

WebServerLib.BaseRequestHandler() # invalid, this is a module!

WebServerLib.BaseRequestHandler.BaseRequestHandler() # this is not intuitive!
```

Whereas using different names avoids the issue:
```py
import WebServerLib.base

WebServerLib.base.BaseRequestHandler() # namespace makes clear what is a module and what is a class
```

With that said, it is true that other `WebServerLib` files are named as `_Handler`. If there is a strong preference for consistency I will make the change. What do you think?

Reply:
Great, thanks for the feedback. Will make the change 👍

Reply:
Updated as `BaseRequestHandler.py` and `BaseRequestHandler` class.